examples/run_largecyc_CPMG.py

```diff
@@ -22,18 +22,18 @@
 If you wish to keep the field as is without adjustment, follow
 the 'py run_CPMG.py' command with 'stayput' (e.g. 'py run_CPMG.py stayput')
 """
-from pyspecdata import getDATADIR,r_
+import pyspecdata as psd
 import os
 import sys
-from numpy import array, pi
+from numpy import array, pi, r_
 import SpinCore_pp
 from SpinCore_pp import prog_plen, get_integer_sampling_intervals, save_data
 from SpinCore_pp.ppg import generic
 from datetime import datetime
 from Instruments.XEPR_eth import xepr
 
 my_exp_type = "ODNP_NMR_comp/CPMG"
-assert os.path.exists(getDATADIR(exp_type=my_exp_type))
+assert os.path.exists(psd.getDATADIR(exp_type=my_exp_type))
```

Comment:
mostly, this is stylistic

but for clarity, we should get `r_` from numpy -- it's in pyspecdata, but just comes from numpy